src/main/java/net/shadowmage/ancientwarfare/structure/block/BlockBrazierEmber.java

```diff
@@ -0,0 +1,230 @@
+import net.minecraft.world.World;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import net.shadowmage.ancientwarfare.core.AncientWarfareCore;
+import net.shadowmage.ancientwarfare.core.util.NBTBuilder;
+
+import javax.annotation.Nullable;
+import java.util.Random;
+
+public class BlockBrazierEmber extends BlockBaseStructure {
+	public static final String LIT_TAG = "lit";
+	private static final PropertyBool LIT = PropertyBool.create(LIT_TAG);
+
+	private static final AxisAlignedBB AABB = new AxisAlignedBB(0D, 0D, 0D, 1D, 0.88D, 1D);
+
+	public BlockBrazierEmber() {
+		super(Material.ROCK, "brazier_ember");
+		setHardness(2);
+		setTickRandomly(true);
+	}
+
+	@Override
+	protected BlockStateContainer createBlockState() {
+		return new BlockStateContainer(this, LIT);
+	}
+
+	@Override
+	public void getSubBlocks(CreativeTabs itemIn, NonNullList<ItemStack> items) {
+		Item item = Item.getItemFromBlock(this);
+			ItemStack stack = new ItemStack(item);
+			stack.setTagCompound(new NBTBuilder().setBoolean(LIT_TAG, false).build());
+			items.add(stack);
+			stack = new ItemStack(item);
+			stack.setTagCompound(new NBTBuilder().setBoolean(LIT_TAG, true).build());
+			items.add(stack);
+	}
+
+	@Override
+	public IBlockState getStateFromMeta(int meta) {
+		return getDefaultState().withProperty(LIT, (meta >> 3) == 1);
+	}
+
+	@Override
+	public int getMetaFromState(IBlockState state) {
+		return (state.getValue(LIT) ? 1 << 3 : 0);
+	}
+
+	@Override
+	public boolean isFullCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public boolean isOpaqueCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public boolean isNormalCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public AxisAlignedBB getBoundingBox(IBlockState state, IBlockAccess source, BlockPos pos) { return AABB; }
+
+	@Nullable
+	@Override
+	public AxisAlignedBB getCollisionBoundingBox(IBlockState blockState, IBlockAccess worldIn, BlockPos pos) {
+		return AABB;
+	}
+
+	@Override
+	public boolean isPassable(IBlockAccess worldIn, BlockPos pos) {
+		return false;
+	}
+
+	@Override
+	public int getLightValue(IBlockState state, IBlockAccess world, BlockPos pos) {
+		return state.getValue(LIT) ? 6 : 0;
+	}
+
+	@Override
+	public BlockFaceShape getBlockFaceShape(IBlockAccess worldIn, IBlockState state, BlockPos pos, EnumFacing face) {
+		return BlockFaceShape.UNDEFINED;
+	}
+
+	@Override
+	public boolean canPlaceBlockAt(World worldIn, BlockPos pos) {
+		return worldIn.getBlockState(pos.down()).isSideSolid(worldIn, pos.down(), EnumFacing.UP);
+	}
+
+	@Override
+	public void onBlockPlacedBy(World worldIn, BlockPos pos, IBlockState state, EntityLivingBase placer, ItemStack stack) {
+		if (!stack.hasTagCompound()) {
+			return;
+		}
+		//noinspection ConstantConditions
+		state = state
+				.withProperty(LIT, stack.getTagCompound().getBoolean(LIT_TAG));
```

Comment:
this property being set on a new line doesn't look great - just removing that new line should work better